.github/workflows/build-dependencies-table.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Dependencies Table
+
+on:
+  release:
+    types: [published]
```

Comment:
Would there be advantages to doing this as part of our PR process for other data model changes? 

We could do it after JSON-LD generation. It might be a useful sniff test. That would require moving this job to be part of the `ci-lint-validate-convert` action and making it dependent on the JSON-LD creation being successful.